test/plugins/features/movies/controller.test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+const Movie      = require('../../../../lib/models/movie');
+
+describe('movie controller', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', () => {
```

Comment:
Might be nice in this test to make sure that the controller also saves the release year